### PR TITLE
Add a public API to set password for an encrypted PDF

### PR DESCRIPTION
**Pull request by srijanshetty**

### CODE REVIEW

**Comment by srijanshetty** [comment]
@modesty could you kindly review this PR and merge it, if possible!

Thanks!

File: lib/pdf.js [hunk 1]
```diff
@@ -252,10 +252,10 @@ let PDFJSClass = (function () {
     };
 
 
-    cls.prototype.parsePDFData = function(arrayBuffer) {
+    cls.prototype.parsePDFData = function(arrayBuffer, password) {
         this.pdfDocument = null;
 
-        let parameters = {password: '', data: arrayBuffer};
+        let parameters = {password: password, data: arrayBuffer};
```
Comment (modesty) [inline lib/pdf.js]: line 255: cls.prototype.parsePDFData = function(arrayBuffer, password: '') {
line 258: let parameters = {password, data: arrayBuffer};

Comment (srijanshetty) [reply lib/pdf.js]: Yes! I'll make the changes

**Comment by srijanshetty** [comment]
Closing this for a preferred feature branch based pull request.

**Pull request closed without merging**